src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            String newRandomBall = lottery.getRandomBall().toString();
+            System.out.println(newRandomBall);
```

Comment:
Use one instance of `Lottery`
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < COUNT_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
```